clients/src/main/java/org/apache/kafka/clients/NodeApiVersions.java

```diff
@@ -233,4 +256,8 @@ public ApiVersion apiVersion(ApiKeys apiKey) {
     public Map<ApiKeys, ApiVersion> allSupportedApiVersions() {
         return supportedVersions;
     }
+
+    public Map<String, SupportedVersionRange> supportedFeatures() {
```

Comment:
We already get a broker's supported features via ApiVersionsResponse, but we were not storing it in NodeApiVersions. This adds that.